scripts/_trap

```diff
@@ -0,0 +1,39 @@
+
```

Comment:
I'm not really a bash expert, I couldn't figure out what this is used for. Could you add 1-2 lines to explain it?

Reply:
Nor am I :D
I borrowed this from another project. From my understanding, it catches the exit codes if any and gives some nice output to the user.

Reply:
Nor am I.
Borrowed this from another project.
I guess it catches the exit codes if any and gives some nice human readable output